APMrover2/sensors.cpp

```diff
@@ -127,67 +127,67 @@ void Rover::accel_cal_update() {
     }
 }
 
-// read the sonars
-void Rover::read_sonars(void)
+// read the rangefinders
+void Rover::read_rangefinders(void)
 {
-    sonar.update();
+    rangefinder.update();
 
-    if (sonar.status(0) == RangeFinder::RangeFinder_NotConnected) {
-        // this makes it possible to disable sonar at runtime
+    if (rangefinder.status(0) == RangeFinder::RangeFinder_NotConnected) {
+        // this makes it possible to disable rangefinder at runtime
         return;
     }
 
-    if (sonar.has_data(1)) {
-        // we have two sonars
-        obstacle.sonar1_distance_cm = sonar.distance_cm(0);
-        obstacle.sonar2_distance_cm = sonar.distance_cm(1);
-        if (obstacle.sonar1_distance_cm < static_cast<uint16_t>(g.sonar_trigger_cm) &&
-            obstacle.sonar1_distance_cm < static_cast<uint16_t>(obstacle.sonar2_distance_cm))  {
+    if (rangefinder.has_data(1)) {
+        // we have two rangefinders
+        obstacle.rangefinder1_distance_cm = rangefinder.distance_cm(0);
+        obstacle.rangefinder2_distance_cm = rangefinder.distance_cm(1);
+        if (obstacle.rangefinder1_distance_cm < static_cast<uint16_t>(g.rangefinder_trigger_cm) &&
+            obstacle.rangefinder1_distance_cm < static_cast<uint16_t>(obstacle.rangefinder2_distance_cm))  {
             // we have an object on the left
             if (obstacle.detected_count < 127) {
                 obstacle.detected_count++;
             }
-            if (obstacle.detected_count == g.sonar_debounce) {
-                gcs().send_text(MAV_SEVERITY_INFO, "Sonar1 obstacle %u cm",
-                        static_cast<uint32_t>(obstacle.sonar1_distance_cm));
+            if (obstacle.detected_count == g.rangefinder_debounce) {
+                gcs().send_text(MAV_SEVERITY_INFO, "rangefinder1 obstacle %u cm",
```

Comment:
Not very important, but maybe better to keep the initial letter uppercase? There's another one below.